app/src/main/java/phuchh/com/music_60/adapter/NowPlayingAdapter.java

```diff
@@ -34,7 +34,8 @@ public ViewHolder onCreateViewHolder(@NonNull ViewGroup viewGroup, int i) {
 
     @Override
     public void onBindViewHolder(@NonNull ViewHolder viewHolder, int i) {
-        viewHolder.setData(i, mTracks.get(i).getTitle());
+        viewHolder.setIsRecyclable(false);
+        viewHolder.setData(String.valueOf(i+1), mTracks.get(i).getTitle());
```

Comment:
```suggestion
        viewHolder.setData(String.valueOf(++i), mTracks.get(i).getTitle());
```

Reply:
để ++i thì giá trị i thay đổi theo nên khi kéo xuống sẽ bị index out of bound ạ

Reply:
Em xem lại coi. a thấy các bạn trước vẫn xử được mà.